strawberry_django_plus/middlewares/user_warmup.py

```diff
@@ -0,0 +1,27 @@
+import inspect
+from typing import Any, Awaitable, Callable, Coroutine, cast
+
+from asgiref.sync import sync_to_async
+from django.http import HttpRequest, HttpResponse
+from django.utils.decorators import sync_and_async_middleware
+
+
+@sync_and_async_middleware
+def user_warmup_middleware(
+    get_response: Callable[[HttpRequest], HttpResponse | Coroutine[Any, Any, HttpResponse]],
+):
+    if inspect.iscoroutinefunction(get_response):
+
+        async def middleware(request):
```

Comment:
This will make the pyright issue go away here

```suggestion
        async def middleware(request):  # type: ignore
```